docusaurus/docs/learn/quickstarts/services/kubernetes-service.md

```diff
@@ -2,69 +2,81 @@
+    ```text
+    ziti edge create identity "hello-client" \
+        --role-attributes hello-clients \
+        --jwt-output-file hello-client.jwt
+    ```
 
-    ziti edge create service-policy "hello-dial-policy" Dial \
-        --service-roles '@hello-service' --identity-roles '#hello-clients'
+1. Ensure you have router policies for your identities and services.
 
-    # adds a role to "miniziti-client" from the local Kubernetes quickstart        
-    ziti edge update identity "miniziti-client" \
-        --role-attributes testapi-clients,hello-clients
+    ```text
+    ziti edge list edge-router-policies
+    ziti edge list service-edge-router-policies
     ```
 
-1. Deploy Hello Toy.
-
-   This chart is a regular, non-OpenZiti demo server deployment. Next we'll connect it to our network with a tunneler deployment.
+    You can create a default router policy for both if needed.
 
     ```text
-    helm install "hello-toy" openziti/hello-toy \
-        --namespace hello-toy --create-namespace \
-        --set serviceDomainName=minihello
+    ziti edge create edge-router-policy "default" \
+    --edge-router-roles '#all' --identity-roles '#all'
+
+    ziti edge create service-edge-router-policy "default" \
+    --edge-router-roles '#all' --service-roles '#all'
     ```
 
-1. Deploy a tunneler Pod.
+1. Create the service.
 
     ```text
-    helm install "ziti-host" openziti/ziti-host \
-        --namespace hello-toy \
-        --set-file zitiIdentity=/tmp/hello-host.json
-    ```
+    ziti edge create config "hello-intercept-config" intercept.v1 \
+        '{"protocols":["tcp"],"addresses":["hello.ziti.internal"], "portRanges":[{"low":80, "high":80}]}'
 
-1. Wait for deployment.
+    ziti edge create config "hello-host-config" host.v1 \
+        '{"protocol":"tcp", "address":"hello.hello-toy.svc","port":80}'
+
+    ziti edge create service "hello-service" \
+        --configs hello-intercept-config,hello-host-config
+
+1. Create the service policies for the client and host identities.
 
     ```text
-    kubectl wait deployments "ziti-host" \
-        --namespace hello-toy \
-        --for condition=Available=True \
-        --timeout=90s
+    ziti edge create service-policy "hello-dial-policy" Dial \
+        --service-roles '@hello-service' --identity-roles '#hello-clients'
+
+    ziti edge create service-policy "hello-bind-policy" Bind \
+        --service-roles '@hello-service' --identity-roles '#hello-hosts'
     ```
 
-1. Visit the Hello Demo page in your browser: [http://minihello.ziti/](http://minihello.ziti/)
+1. Deploy the Hello Toy chart.
 
-   Now you have two services available to your tunneler:
-   * hello-service
-   * testapi-service
+    ```text
+    helm install "hello-toy" openziti/hello-toy \
+        --namespace hello-toy --create-namespace \
+        --set serviceDomainName=hello
+    ```
 
+1. Visit the Hello Demo page in your browser: [http://hello.ziti.internal/](http://hello.ziti.internal/)
```

Comment:
you skipped a step to install an identity?

Reply:
It's on line 27 (attempted diff link: https://github.com/openziti/ziti-doc/pull/873/files#diff-3ff9ef69e1e2c17b10e58b3eff919a73ad8b3ef5beb557cc0a3b39e6f7ddae17R27)

Reply:
nope - i missed it when reviewing. i see it up there now